pkgs/tools/networking/mitmproxy/default.nix

```diff
@@ -79,5 +79,6 @@ buildPythonPackage rec {
     homepage    = "https://mitmproxy.org/";
     license     = licenses.mit;
     maintainers = with maintainers; [ fpletz kamilchm ];
+    broken = true;
```

Comment:
mitmproxy is not at fault for the failure of its `TestMasterSecretLogger.test_log` on release-20.03. It does not fail [on master](https://hydra.nixos.org/jobset/nixpkgs/trunk/jobs-tab?filter=mitmproxy), yet there is no difference between mitmproxy definitions between release-20.03 and master.